command/repo.go

```diff
@@ -89,6 +92,20 @@ With '--web', open the repository in a web browser instead.`,
 	RunE: repoView,
 }
 
+var repoCreditsCmd = &cobra.Command{
+	Use:   "credits [repository]",
```

Comment:
Nit: since repository is [a placeholder](https://github.com/cli/cli/blob/trunk/docs/command-line-syntax.md), this should be `[<repository>]`